scripts/generate_json_docs.py

```diff
@@ -294,12 +290,14 @@ def build_type(type_id, title, contents):
     }
 
 
-def clean_source_path(source):
-    source_path = ''
-    if ABSOLUTE_LIBRARY_PATH in source:
-        source_path = source.replace(ABSOLUTE_LIBRARY_PATH, source)
-
-    return source_path
+def clean_source_path(module):
+    if isinstance(module, str):
```

Comment:
Instead of `str` use `six.text_type`, `six.binary_type` or `six.string_types` depending on what you are going for.